src/pages/faq/comparison/terraform.mdx

```diff
@@ -0,0 +1,183 @@
+export const description = 'Comparison to Terraform'
+
+# Terraform vs. Nitric
+
+<Note>
+  Nitric is not designed to replace IaC tools like Terraform but instead
+  introduces a method of bringing developer self-service for infrastructure
+  directly into the developer application (Nitric's default deployment engines
+  are built with Pulumi). Nitric can be augmented through use of tools like
+  Pulumi and even be fully customized using such tools see [Custom
+  Providers](/docs/reference/providers/custom/building-custom-provider) for more
+  details.
+</Note>
+
+Nitric is a framework designed to aid developers in building full cloud applications, including infrastructure. [Terraform](https://www.terraform.io/) is an Infrastructure as Code (IaC) tool that enables explicit definition of infrastructure using HCL (or programming languages using [CDKTF](https://developer.hashicorp.com/terraform/cdktf)). The main differences between these are:
+
+1. Nitric is cloud-agnostic, code that is written using Nitric constructs can be deployed to any cloud. Terraform supports many clouds, infrastructure declarations are explicitly defined for the provider that a resource is provided by.
+
+2. Nitric defines not only the infrastructure but how it is interacted with at runtime, so infrastructure can be automatically inferred from application code to ensure best practice deployments and least privilege security.
+
+3. Nitric provides tools for locally simulating a cloud environment (using the Nitric CLI), to allow an application to be tested locally. Pulumi programs can be unit tested but applications written to be deployed will need a separate solution for local development such as LocalStack.
```

Comment:
Says "Pulumi programs can be unit tested..." should this talk about Terraform testing/local development options instead?

Reply:
fixed, and added notes on linting/validation/contract testing for terraform